### PR TITLE
Fix Chinese translation

### PR DESCRIPTION
**Pull request by Enaium**
Fix Chinese translation

### CODE REVIEW

File: src/main/resources/translations/zh-cn.json [hunk 1]
```diff
@@ -375,5 +379,252 @@
...
+	"binding.inputprompt.initial": " <waiting>",
+	"binding.inputprompt.finish": " <ENTER to finish>",
+	"binding.close.window.name": "关闭窗口",
+	"binding.close.window.desc": "关闭当前活动窗口.",
+	"binding.close.tab.name": "关闭标签",
+	"binding.close.tab.desc": "关闭当前的活动标签.",
+	"binding.saveapp.name": "保存应用程序",
+	"binding.saveapp.desc": "将应用程序保存到文件.",
+	"binding.save.name": "保存更改",
+	"binding.save.desc": "在当前编辑器视口中应用更改.",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:481]:
```suggestion
	"binding.save.desc": "保存当前编辑器窗口中的更改.",
```

File: src/main/resources/translations/zh-cn.json [hunk 2]
```diff
@@ -375,5 +379,252 @@
...
+	"binding.undo.name": "撤消更改",
+	"binding.undo.desc": "撤消当前编辑器视口中最近的更改.",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:483]:
```suggestion
	"binding.undo.desc": "撤消当前编辑器窗口中最近的更改.",
```

File: src/main/resources/translations/zh-cn.json [hunk 3]
```diff
@@ -375,5 +379,252 @@
...
+	"decompile.showsynthetics.desc": "编译器生成的代码",
+	"decompile.timeout.name": "超时",
+	"decompile.timeout.desc": "中止反编译进程之前等待的时间（以毫秒为单位）",
+	"decompile.fail": "反编译失败。 您可以在配置菜单中更改反编译器.",
+
+	"assembler": "反编译器",
+	"assembler.verify.name": "校验",
+	"assembler.verify.desc": "校验方法字节码。 这也可以更好地进行变量类型分析.",
+	"assembler.variables.name": "变量",
+	"assembler.variables.desc": "Save variable debug information after assembling.",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:506]:
```suggestion
	"assembler.variables.desc": "保存时写入变量调试信息.",
```

File: src/main/resources/translations/zh-cn.json [hunk 4]
```diff
@@ -375,5 +379,252 @@
...
+
+	"suggest.decompile.warn.title": "反编译有错误",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:508]:
```suggestion
	"suggest.decompile.warn.title": "反编译结果出错",
```

File: src/main/resources/translations/zh-cn.json [hunk 5]
```diff
@@ -375,5 +379,252 @@
...
+	"suggest.decompile.warn": "当前反编译器的输出有一些问题.\n这将禁用类中的交互.\n要切换到另一个反编译器吗?",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:509]:
```suggestion
	"suggest.decompile.warn": "当前反编译器的输出存在问题.\n这将禁用交互功能.\n要切换到另一个反编译器吗?",
```

File: src/main/resources/translations/zh-cn.json [hunk 6]
```diff
@@ -1,100 +1,93 @@
...
+	"ui.history": "历史",
+	"ui.history.pop": "退回到上一个状态",
+	"ui.history.open.class": "打开类",
+	"ui.history.open.file": "打开文件",
+	"ui.info.logging": "日志",
+	"ui.info.other": "其他",
+	"ui.attach": "注入",
+	"ui.attach.copy": "复制到工作空间",
+	"ui.noerrormsg": "未指定错误消息",
+	"ui.openissue": "如果您认为这是一个错误，请报告此异常",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:27]:
```suggestion
	"ui.openissue": "如果您认为这是BUG，请报告此异常",
```

File: src/main/resources/translations/zh-cn.json [hunk 7]
```diff
@@ -1,100 +1,93 @@
...
+	"ui.config": "配置",
+	"ui.history": "历史",
+	"ui.history.pop": "退回到上一个状态",
+	"ui.history.open.class": "打开类",
+	"ui.history.open.file": "打开文件",
+	"ui.info.logging": "日志",
+	"ui.info.other": "其他",
+	"ui.attach": "注入",
+	"ui.attach.copy": "复制到工作空间",
+	"ui.noerrormsg": "未指定错误消息",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:26]:
```suggestion
	"ui.noerrormsg": "无错误消息",
```
这里的意思应该是抛出的 exception 没有 message 部分。

File: src/main/resources/translations/zh-cn.json [hunk 8]
```diff
@@ -1,100 +1,93 @@
...
 	"ui.menubar.file.agentexport": "应用更改",
 	"ui.menubar.config": "配置",
+	"ui.menubar.themeeditor": "主题编辑器",
 	"ui.menubar.search": "搜索",
+	"ui.menubar.search.string": "字符串",
+	"ui.menubar.search.value": "数值",
+	"ui.menubar.search.cls_reference": "Class引用",
+	"ui.menubar.search.mem_reference": "Member引用",
+	"ui.menubar.search.declare": "声明",
+	"ui.menubar.search.insn": "说明",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:61]:
```suggestion
	"ui.menubar.search.insn": "指令",
```

File: src/main/resources/translations/zh-cn.json [hunk 9]
```diff
@@ -1,100 +1,93 @@
...
 	"ui.edit.tab.fields": "字段",
-	"ui.edit.tab.fields.flags": "修饰符",
+	"ui.edit.tab.fields.access": "访问",
 	"ui.edit.tab.fields.name": "名称",
 	"ui.edit.tab.fields.type": "类型",
-	"ui.edit.method.define": "显示定义",
+	"ui.edit.tab.fields.index": "索引",
+	"ui.edit.method.rename": "重命名",
+	"ui.edit.method.follow": "跟随",
+	"ui.edit.method.referrers": "推荐",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:89]:
```suggestion
	"ui.edit.method.referrers": "来源",
```

File: src/main/resources/translations/zh-cn.json [hunk 10]
```diff
@@ -1,100 +1,93 @@
...
+	"ui.edit.method.goto": "转到定义",
+	"ui.edit.method.define": "开放定义",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:91]:
```suggestion
	"ui.edit.method.define": "打开定义",
```

File: src/main/resources/translations/zh-cn.json [hunk 11]
```diff
@@ -375,5 +379,252 @@
...
+	"asm.out.computeframes.name": "计算框架",
+	"asm.out.computeframes.desc": "强制ASM计算堆栈框架.",
+	"asm.out.reflectionexport.name": "反射导出",
+	"asm.out.reflectionexport.desc": "允许导出进程从运行时查找丢失的类.",
+	"asm.in.skipcode.name": "跳过代码",
+	"asm.in.skipcode.desc": "强制ASM跳过方法代码.",
+	"asm.in.skipdebug.name": "跳过调试",
+	"asm.in.skipdebug.desc": "强制ASM跳过调试信息，如变量名",
+	"asm.in.skipframes.name": "跳过框架",
+	"asm.in.skipframes.desc": "加载新输入时强制ASM跳过现有堆栈框架.",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:424]:
```suggestion
	"asm.in.skipframes.desc": "强制ASM跳过已有的 frame 信息.",
```

File: src/main/resources/translations/zh-cn.json [hunk 12]
```diff
@@ -375,5 +379,252 @@
...
+	"binding.rename.desc": "重命名选定的类或成员.",
+	"binding.gotodef.name": "转到定义",
+	"binding.gotodef.desc": "跳转到选定项的定义.",
+
+	"backend": "Backend / Private",
+
+	"decompile": "反编译",
+	"decompile.decompiler.name": "反编译器",
+	"decompile.decompiler.desc": "要使用的反编译器引擎",
+	"decompile.showsynthetics.name": "显示synthetics",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:496]:
```suggestion
	"decompile.showsynthetics.name": "显示 synthetics 成员",
```

File: src/main/resources/translations/zh-cn.json [hunk 13]
```diff
@@ -375,5 +379,252 @@
...
+	"binding.rename.name": "重命名",
+	"binding.rename.desc": "重命名选定的类或成员.",
+	"binding.gotodef.name": "转到定义",
+	"binding.gotodef.desc": "跳转到选定项的定义.",
+
+	"backend": "Backend / Private",
+
+	"decompile": "反编译",
+	"decompile.decompiler.name": "反编译器",
+	"decompile.decompiler.desc": "要使用的反编译器引擎",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:495]:
```suggestion
	"decompile.decompiler.desc": "要使用的反编译器.",
```

File: src/main/resources/translations/zh-cn.json [hunk 14]
```diff
@@ -375,5 +379,252 @@
...
+	"decompile.showsynthetics.name": "显示synthetics",
+	"decompile.showsynthetics.desc": "编译器生成的代码",
```
Comment (andylizi) [inline src/main/resources/translations/zh-cn.json:497]:
```suggestion
	"decompile.showsynthetics.desc": "显示编译器生成的代码.",
```

File: src/main/resources/translations/zh-cn.json [hunk 15]
```diff
@@ -375,5 +379,252 @@
...
+	"ui.bean.typeannotation.reftype.unknown": "未知",
+
+
+
+	"update.fail.resolve": "无法获取自身可执行文件的路径.",
+	"update.fail.jarread": "无法读取内置的 pom.xml.",
+	"update.fail.nodownload": "Recaf 有新版本, 但获取不到下载地址.",
+	"update.fail.unknwon": "更新检查失败: ",
+	"update.outdated": "Recaf 有新版本",
+	"update.complete": "更新完成,请运行新版本.",
```
[the remark this answers is not in the document]
Comment (andylizi) [reply src/main/resources/translations/zh-cn.json:396]: 唔注意到还有其他很多地方逗号/句号后无空格。

**Comment by Col-E** [comment]
A lot of the lines in the translation file are actually unused in 2.0. I feel kinda bad that you did more work than you had to. Anyways thanks for this! Let me know when you are ready for this to merge 👍

**Comment by Enaium** [comment]
Can be merged

**Pull request merged**